python/pyspark/pandas/base.py

```diff
@@ -42,7 +42,7 @@
     TimestampType,
 )
 
-from pyspark import pandas as pp  # For running doctests and reference resolution in PyCharm.
+from pyspark import pandas as ps  # For running doctests and reference resolution in PyCharm.
```

Comment:
Is this the proposal of this PR?
cc @HyukjinKwon

Reply:
I am actually working on it in https://github.com/apache/spark/pull/32103 ...